app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError(
+            "Invalid command format. Use: cp <source_file> <destination_file>"
+        )
+    source_file, destination_file = parts[1], parts[2]
+
+    if source_file == destination_file:
+        return
+
+    with (open(source_file, "r") as file_in,
+          open(destination_file, "w") as file_out):
```

Comment:
The code correctly uses double quotes for file operations, which aligns with the code style guideline preferring double quotes.